nekoyume/Assets/_Scripts/UI/Widget/Popup/MaterialNavigationPopup.cs

```diff
@@ -139,8 +139,11 @@ private static void InvokeAfterActionPointCheck(System.Action action)
             _callback = callback;
             itemImage.sprite = itemIcon;
             itemNameText.text = itemName;
-            var split = itemCount.Split('.');
-            itemCountText.text = L10nManager.Localize("UI_COUNT_FORMAT", split[0]);
+
+            if (itemCount.Contains(".") && float.TryParse(itemCount, out var floatCount))
+                itemCount = $"{floatCount:0.####}";
+            itemCountText.text = L10nManager.Localize("UI_COUNT_FORMAT", itemCount);
```

Comment:
```suggestion
            itemCountText.text = L10nManager.Localize("UI_COUNT_FORMAT", itemCount);
```
그냥 itemCount를 split하거나 하지 않고 바로 쓰면 안되나요?

Reply:
소수점이 엄청 길어지는 상황이 생기고 이것이 보기 안 좋을 수 있어서 4자리로 제한을 뒀습니다

Reply:
어프루브를 누르긴 했는데, 생각해보니 float 말고 double이나 decimal로 TryParse 하는게 좋겠습니다. 여기 들어갈 값들이면 전부 원본 값이 long 아니면 BigInteger일거라서...

Reply:
아래 현님의 사례처럼 소수점이 길어지는 상황이 생기더라도 이 팝업의 역할이 재화의 값을 자세히 표시하는 거라 itemCount 그대로 사용해도 되지 않을까 싶긴 합니다. 재화 값이 그렇게 극단적으로 길어지는 경우가 거의 없기도 하구요...